workflow/airflow_libero/Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
not sure where they come from, but `numpy` and `pandas` are inside the set of dependencies, adding >100 MB to the virtualenv size

Reply:
They are dependencies it seems?
https://github.com/apache/incubator-airflow/blob/master/setup.py#L247

Reply:
It's an absolute beast when it comes to dependencies. Once you run a full setup (celery, postgres, redis + other libs) it gets pretty big. But it does try and integrate with a lot of things.